### PR TITLE
OSS standardization changes

### PR DESCRIPTION
**Pull request by emmyoop**
resolves https://github.com/dbt-labs/dbt-core/issues/9221

Description
add `CONTRIBUTING.md` and dependabot alerts. Modify readme to bring this OSS repo up to the minimal standards for OSS at dbt Labs.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+4. Push changes to your fork
+5. Open a pull request against `dbt-labs/dbt-extractor` from your forked repository
+
+### dbt Labs contributors
+
+If you are a member of the `dbt-labs` GitHub organization, you will have push access to the `dbt-extractor` repo. Rather than forking `dbt-extractor` to make your changes, just clone the repository, check out a new branch, and push directly to that branch.
+
+## Setting up an environment
+
+TBD
```
Comment (emmyoop) [inline CONTRIBUTING.md:50]: More details needed here.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+
+### Test commands
+
+```
+make test
+```
+
+## Debugging
+
+TBD
```
Comment (emmyoop) [inline CONTRIBUTING.md:85]: Any debugging tips?

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+
+### Assorted development tips
+
+TBD
```
Comment (emmyoop) [inline CONTRIBUTING.md:89]: Any general development tips?

**Review by QMalcolm — APPROVED** [review]
🚀 Thank you again!

**Pull request merged**